capeinfra/swimlanes/private.py

```diff
@@ -38,6 +38,17 @@ def __init__(self, name, *args, **kwargs):
             opts=ResourceOptions(parent=self),
         )
 
+        subnet_name = f"{name}-vm-subnet"
+        self.vm_subnet = aws.ec2.Subnet(
+            subnet_name,
+            cidr_block="10.0.1.0/24",
```

Comment:
this kinda feels it should be configurable. i won't hold up the merging, but pls make an issue to get that modded

Reply:
Yeah, I'll open an issue for this